### ISSUE TITLE
[Help] Not easy to balance the data when adding new disk to BE

### ISSUE THREAD

**Issue by morningman** [apache/doris#3460]
**Is your feature request related to a problem? Please describe.**
When adding new disks to BE, the data can not do automatic balance among disks within the BE.

**Describe the solution you'd like**
Give a way to balance the data among disks within the BE

**Describe alternatives you've considered**
Utilize the decommission operation. The decommission operation will move all data to other
node. And then the decommission BE is empty, which is like a newly added node.
Then the balance logic can try to balance the data back to that BE, and this will help 
redistributing data evenly to all disks of that BE.

The problem is after decommission, the BE will be dropped automatically, so we should find a
way to NOT drop it.

**Comment by kangkaisen** [comment]
@morningman Hi, Intuitively,I think utilize the decommission operation to solve the balance issue is strange.

**Comment by morningman** [comment]
> @morningman Hi, Intuitively,I think utilize the decommission operation to solve the balance issue is strange.

Yes, So I add a TODO in the comment of the PR #3461. This just a temporary solution.
And by the way, decommission without dropping the BE is also good for some scenarios.

For example, I want to repair a machine offline, but I don't want (or can't) perform an ADD or DROP backend operation, which will changing the topology information of the cluster.

**Comment by kangkaisen** [comment]
> > @morningman Hi, Intuitively,I think utilize the decommission operation to solve the balance issue is strange.
> 
> Yes, So I add a TODO in the comment of the PR #3461. This just a temporary solution.
> And by the way, decommission without dropping the BE is also good for some scenarios.
> 
> For example, I want to repair a machine offline, but I don't want (or can't) perform an ADD or DROP backend operation, which will changing the topology information of the cluster.

OK
